libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat

```diff
@@ -81,10 +83,10 @@ PC5 VBUS INPUT OPENDRAIN
 # Overwise UART1 or UART6 or RCIN will not work
 PB8 TIM4_CH3 TIM4 RCININT PULLDOWN LOW
 
-#Omnibus F4 V3 and later had hw inverter on UART6
+#Omnibus F4 AIO V3 and later had hw inverter on UART6
 #Overide it to use as GPS UART port
-PC8 SBUS_INVERT_RX OUTPUT LOW
-PC9 SBUS_INVERT_TX OUTPUT LOW
+#PC8 SBUS_INVERT_RX OUTPUT LOW
```

Comment:
If we split into different targets, it is better to remove this, not to comment.

Reply:
will make those changes....am using the "include" method for the OmnibusF4 board....but before I push the changes, I want to try to see if I can make the board LED actually show the arming status, currently, it flashes during the bootloader , then is inactive from then on...